server_admin/topics/user-federation/ldap.adoc

```diff
@@ -32,6 +31,13 @@ The benefit to this approach is that you do not have the overhead of importing a
 
 This storage mode is controled by the `Import Users` switch.  Set to `On` to import users.
 
+NOTE: In the case that import of users is disabled, there is no support to save user profile attributes and most of other user metadata
+      into the {project_name} database. The single exception to this are user profile metadata, which are mapped to the LDAP. This possibly includes
```

Comment:
Try this:
NOTE: If user import is disabled, you cannot save user profile attributes into the {project_name} database. Also you cannot save metadata except for user profile metadata that are mapped to the LDAP.

Reply:
I will change this exactly as you suggested.